extensions/skins/conversation_v1/player.html

```diff
@@ -36,30 +34,22 @@
         cards, (b) the screen is wide, in which case both cards appear
         side-by-side.
       -->
-      <md-card class="conversation-skin-tutor-card conversation-skin-tutor-card-active"
-               ng-class="{'conversation-skin-left-card': isCurrentSupplementalCardNonempty() && canWindowFitTwoCards(), 'conversation-skin-has-subsidiary-cards': isCurrentSupplementalCardNonempty() && !canWindowFitTwoCards()}" ng-show="isPanelVisible(PANEL_TUTOR)">
-        <div class="conversation-skin-tutor-card-feedback-toggle-container" ng-if="activeCard.stateName && !isInPreviewMode">
-          <button class="conversation-skin-tutor-card-feedback-toggle"
```

Comment:
OK, makes sense. Let's do the metadata thing in a separate pass.

For the speech thing, could you take a look at #1080? This is actively being worked on and it'd be useful to know the intended behaviour.